qt/plugins/systrayiconplugin.py

```diff
@@ -72,7 +72,7 @@ def init(self, snapshots):
 
         try:
 
-            if tools.is_Qt6_working(systray_required=True):
+            if tools.is_Qt_working(systray_required=True):
```

Comment:
That was the initial reasons for this PR.
That name error appeared in DEBUG output. Then I started to investigate why PyLint can not see that error, too.